Modules/Devices/src/Devices.Infrastructure/Persistence/Database/DevicesDbContext.cs

```diff
@@ -147,6 +147,8 @@ protected override void ConfigureConventions(ModelConfigurationBuilder configura
             .HaveMaxLength(IdentityAddress.MAX_LENGTH).HaveConversion<IdentityAddressValueConverter>();
         configurationBuilder.Properties<DeviceId>().AreUnicode(false).AreFixedLength()
             .HaveMaxLength(DeviceId.MAX_LENGTH).HaveConversion<DeviceIdValueConverter>();
+        configurationBuilder.Properties<CommunicationLanguage>().AreUnicode().AreFixedLength()
```

Comment:
I don't think that there are any language codes that require unicode. Please verify.

```suggestion
        configurationBuilder.Properties<CommunicationLanguage>().AreUnicode(false).AreFixedLength()
```